client/src/pages/FormNewProduct.js

```diff
@@ -1,11 +1,88 @@
 import styled from "styled-components";
 
+//components
+import Container from "../components/Container";
+import FlexboxRow from "../components/FlexboxRow";
 import Toggle from "../components/IconToggle";
+import Button from "../components/Button";
+
+//assets
+import add_image from "../assets/images/add_image.svg";
 
 export default function FormNewProduct() {
   return (
-    <>
-      <input type="text" placeholder="gadg-name" />
-    </>
+    <Container>
+      <H2> add a new product </H2>
+      <FlexboxRow>
+        <InputName type="text" placeholder="gadg-name" />
+        <Toggle />
+      </FlexboxRow>
+      <AddImg src={add_image} />
+      <InputText type="text" placeholder="please describe your gadg" rows="5" />
+      <Flexbox>
+        <Label for="choose_category">
+          gadg-category
+          <Choice id="choose_category">
+            <option value="snow">snow</option>
+            <option value="bike">bike</option>
+            <option value="tool">tool</option>
+            <option value="car">car</option>
+          </Choice>
+        </Label>
+        <Label for="choose_size">
+          gadg-size
+          <Choice id="choose_size">
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+          </Choice>
+        </Label>
+      </Flexbox>
+      <InputText type="text" placeholder="facts about your gadg" rows="2" />
+      <InputText
+        type="text"
+        placeholder="personal preferences for usage"
+        rows="2"
+      />
```

Comment:
in the Inputs your can give a maxLength to **prevent the user** to spam the input field with unnecessary text 😄